testData/inspections/suppression/fix/innerVariableDeclarationSuppressionFix2-after-highlighting.go

```diff
@@ -1,6 +1,6 @@
 package newmath
 
 func TestFunction() {
-	//noinspection GoUnusedVariable
```

Comment:
Is there a way to maintain this while changing the inspection name? I know this is a breaking change.